src/app/beneficiaries/fetch.js

```diff
@@ -1,12 +1,17 @@
 import axios from 'axios'
-export async function fetchDataBeneficiaries() {
+export async function fetchDataBeneficiaries(limit, offset) {
 	const BASEURL = process.env.NEXT_PUBLIC_BASE_URL
-	try{
-		const beneficiaries = await axios.get(
-			`${BASEURL}/acat/patient/`)
-		return beneficiaries.data
-	}
-    catch (error) {
+	try {
+		if (limit !== null) {
+			const beneficiaries = await axios.get(
+				`${BASEURL}/acat/patient?limit=${limit}&offset=${offset}`
+			)
+			return beneficiaries.data
+		} else {
```

Comment:
shouldn't this take in account a possible offset null value and assign a default one?

Reply:
Refactored 👍🏻